content/accessibility/index.mdx

```diff
@@ -506,7 +506,7 @@ A third method that can be use to include instructional information from outside
 
 ### Required Fields
 
-It is common to denote required fields with a different color, asterisk, other visual treatment, but this is insufficient for non-sighted users and those with color blindness. To ensure screen-reader support use the `aria-required=”true”` attribute in your form field. When the form field has focus it will announce “required”. Do **not** use the HTML5 `required` attribute. It is not fully supported by screen-readers and is not easily styled with css.
+It is common to denote required fields with a different color, asterisk, or other visual treatment, but this is insufficient for non-sighted users and those with color blindness. To ensure screen-reader support use the `aria-required=”true”` attribute in your form field. When the form field has focus it will announce “required”. Do **not** use the HTML5 `required` attribute. It is not fully supported by screen-readers and is not easily styled with css.
```

Comment:
Suggestion: change treatment to be plural or change 'other' to 'another'
1. 'or `another` visual treatment'
2. 'or other visual `treatments`'